.dev/docker/README.md

```diff
@@ -0,0 +1,69 @@
+# Pterodactyl Panel - Docker Image
+This is a ready to use docker image for the panel.
+
+## Requirements
+This docker image requires some additional software to function. The software can either be provided in other containers (see the [docker-compose.yml](docker-compose.yml) as an example) or as existing instances.
+
+A mysql database is required. We recommend [this](quay.io/parkervcp/mariadb-alpine) image if you prefer to run it in a docker container. As a non-containerized option we recommend mariadb.
```

Comment:
I can't open that link, 403 unauthorized

Reply:
I forgot about this. Moving to use the stock mariadb image anyways.